README.md

```diff
@@ -75,7 +75,7 @@ repo](https://github.com/rapidsai/notebooks-contrib).
 | | Truncated Singular Value Decomposition (tSVD) | Multi-GPU version available (CUDA 10 only) |
 | | Uniform Manifold Approximation and Projection (UMAP) | |
 | | Random Projection | |
-| | t-Distributed Stochastic Neighbor Embedding (TSNE) | (Experimental) |
+| | t-Distributed Stochastic Neighbor Embedding (TSNE) | Both Barnes-Hut and Exact algorithms are provided |
```

Comment:
Removing experimental is good but I wouldn't add the extra detail because we don't provide that level of detail elsewhere.

Reply:
Ohh ok cool :)